src/lib/openapi/spec/api-token-schema.ts

```diff
@@ -6,47 +6,72 @@ export const apiTokenSchema = {
     type: 'object',
     additionalProperties: false,
     required: ['username', 'type'],
+    description: 'An overview of an API token.',
```

Comment:
Maybe we could link to the docs here?

```suggestion
    description: 'An overview of an [Unleash API token](https://docs.getunleash.io/reference/api-tokens-and-client-keys).',
```

Reply:
Yeah absolutely!